frontend/src/APIClients/AuthAPIClient.ts

```diff
@@ -23,20 +25,17 @@ const login = async (
     );
     return data;
   } catch (error) {
-    return null;
-  }
-};
-
-const loginWithGoogle = async (idToken: string): Promise<LoginResponse> => {
-  try {
-    const { data } = await baseAPIClient.post(
-      "/auth/login",
-      { idToken },
-      { withCredentials: true },
-    );
-    return data;
-  } catch (error) {
-    return null;
+    const axiosErr = (error as any) as AxiosError;
+    if (axiosErr.response && axiosErr.response.status === 401) {
+      return {
+        errCode: axiosErr.response.status,
+        errMessage: getLoginErrMessage(axiosErr.response)
```

Comment:
So the main issue with not parsing error messages with the updated implementation is that we'd end up displaying _any_ Firebase-error message (that would have otherwise been caught with "Error logging in. Please try again later.") ; I think for that reason preserving the parsing is better